dxr/plugins/clang/condense.py

```diff
@@ -152,6 +152,16 @@ def _process_loc(locstring):
     return src, Position(row, col)
 
 
+def process_maybe_impl(parents, children, props):
```

Comment:
I'd find docstrings really helpful on these "maybe" functions, since the names aren't the clearest.